ui/src/app/guards/registration-guard.service.ts

```diff
@@ -33,18 +32,12 @@ export class RegistrationGuard implements CanActivate, CanActivateChild {
           return Observable.from([true]);
         }
         return this.profileStorageService.profile$.flatMap((profile) => {
-          // TODO: Remove this logic when enableComplianceLockout feature goes in
-          //   Since dataAccessLevel is computed using the the training and eraCommons modules,
-          //   for now recalculate it using only betaAccess and emailVerification.
-          const hasAccess = environment.enableComplianceLockout ?
-              hasRegisteredAccess(profile.dataAccessLevel) :
-              (!!profile.betaAccessBypassTime &&
-                profile.emailVerificationStatus === 'subscribed');
-          if (hasAccess) {
+          if (hasRegisteredAccess(profile.dataAccessLevel)) {
             return Observable.from([true]);
+          } else {
```

Comment:
Nit: you don’t need the else clause because the return above will short circuit.

Reply:
Ack. My personal aesthetic has always been for evenly-balanced if/else statements in situations like this (where the first return isn't obviously a "return early" kind of scenario). But if there's a strong codebase style in the other direction, I'll be happy to follow-up with a change.

Reply:
Not strong enough that I think it’s something super important to do. I remember getting the suggestion at some point a while ago, but explicit else seems fine.